cms/djangoapps/contentstore/views/videos.py

```diff
@@ -37,39 +37,40 @@
 
 class StatusDisplayStrings(object):
     """
-    Enum of display strings for Video Status presented in Studio (e.g., in UI and in CSV download).
+    A class to map status strings as stored in VAL to display strings for the
+    video upload page
     """
+
     # Translators: This is the status of an active video upload
-    UPLOADING = _("Uploading")
+    _UPLOADING = ugettext_noop("Uploading")
     # Translators: This is the status for a video that the servers are currently processing
-    IN_PROGRESS = _("In Progress")
+    _IN_PROGRESS = ugettext_noop("In Progress")
     # Translators: This is the status for a video that the servers have successfully processed
-    COMPLETE = _("Complete")
+    _COMPLETE = ugettext_noop("Complete")
     # Translators: This is the status for a video that the servers have failed to process
-    FAILED = _("Failed"),
+    _FAILED = ugettext_noop("Failed"),
     # Translators: This is the status for a video for which an invalid
     # processing token was provided in the course settings
-    INVALID_TOKEN = _("Invalid Token"),
+    _INVALID_TOKEN = ugettext_noop("Invalid Token"),
```

Comment:
A simple fix to this i8n issue would be to just change these into instance methods rather than class variables.  Such a change may be simpler for code readability.